tests/utils/test_inference.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Test inference APIs.
+"""
+import os
+import tempfile
+from functools import partial
+
+import pytest
+import torch
+
+from composer.models import composer_resnet
+from composer.utils import export_for_inference
+
+
+@pytest.mark.timeout(10)
+@pytest.mark.parametrize(
+    'model_cls, sample_input',
+    [
+        (partial(composer_resnet, 'resnet18'), (torch.rand(4, 3, 224, 224), torch.randint(10, (4,)))),
+    ],
+)
+def test_export_for_inference_torchscript(model_cls, sample_input):
+    model = model_cls()
+    model.eval()
+
+    orig_out = model(sample_input)
+
+    tempdir_ctx = tempfile.TemporaryDirectory()
+    target_format = 'torchscript'
+    with tempdir_ctx as tempdir:
```

Comment:
Super minor, but can this just be:
```suggestion
    target_format = 'torchscript'
    with tempfile.TemporaryDirectory() as tempdir:
```
When I see a tempfile call outside of a `with` statement I immediately skip ahead in the function looking for where it gets cleaned up, whereas it's immediately clear that it does get cleaned up when instantiated directly in a `with` statement. Also just saves a line.

(applies in a couple other places as well)

Reply:
Fixed it here and at a couple of other places.